package.json

```diff
@@ -93,6 +93,7 @@
     "format": "prettier --write \"**/*.{js,jsx,json,md}\"",
     "generate-heading-ids": "node src/scripts/generate-heading-ids.js",
     "merge-translations": "node src/scripts/merge-translations.js",
+	"check-markdown-language-tag": "node src/scripts/checkMarkdownLanguage.mjs",
```

Comment:
This is a JS file and not a TS file. I can write (and would prefer) typescript but, I didn't want to add the compilation step.